mrp_subcontracting_skip_no_negative/models/stock_move.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2023 Quartile Limited
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import models
+
+
+class StockMove(models.Model):
+    _inherit = "stock.move"
+
+    def _action_done(self, cancel_backorder=False):
+        moves = self.filtered(lambda x: not x.is_subcontract)
+        res = super(StockMove, moves)._action_done(cancel_backorder=cancel_backorder)
+        for move in self - moves:
+            if move.is_subcontract:
+                super(
+                    StockMove, move.with_context(skip_negative_qty_check=True)
+                )._action_done(cancel_backorder)
+        return res
```

Comment:
All the processed moves should be returned if I'm not mistaken. Can you please double-check this works OK.
```suggestion
        subcontract_moves = self.filtered(lambda x: x.is_subcontract).with_context(skip_negative_qty_check=True)
        res = super(StockMove, moves - subcontract_moves)._action_done(cancel_backorder=cancel_backorder)
        res += super(StockMove, subcontract_moves)._action_done(cancel_backorder=cancel_backorder)
        return res
```